src/main/java/bio/terra/janitor/app/configuration/TrackResourcePubsubConfiguration.java

```diff
@@ -0,0 +1,43 @@
+package bio.terra.janitor.app.configuration;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.stereotype.Component;
+
+/** Pub/sub Configurations for tracking resource. */
+@Component
+@Configuration
+@EnableConfigurationProperties
+@ConfigurationProperties(prefix = "pubsub.track-resource")
```

Comment:
let's be consistent and continue to camelCase, so `pubsub.trackResource`

Reply:
Those are prefixes, and for the prefixes, it need to be with dashes. I got compliants if using mixed of lower$upper case as prefix.